manager/orchestrator/global.go

```diff
@@ -192,13 +192,20 @@ func (g *GlobalOrchestrator) reconcileOneService(ctx context.Context, service *a
 		log.G(ctx).WithError(err).Errorf("global orchestrator: reconcileOneService failed finding tasks")
 		return
 	}
+	// indicator if service mode was changed to global
+	modeChanged := false
 	// a node may have completed this service
 	nodeCompleted := make(map[string]struct{})
 	// nodeID -> task list
 	nodeTasks := make(map[string][]*api.Task)
 
 	for _, t := range tasks {
 		if isTaskRunning(t) {
+			// tasks from global service has Slot number 0
```

Comment:
It's a short cut. I check `Slot` number because current orchestrator doesn't cache service before change. Orchestrator should have a diff to get what has changed.